clojure-ts-mode.el

```diff
@@ -726,6 +733,14 @@ literals with regex grammar."
   "Return non-nil if NODE is a Clojure list."
   (string-equal "list_lit" (treesit-node-type node)))
 
+(defun clojure-ts--anon-fn-node-p (node)
```

Comment:
E.g. this can be "anon-fn-lit" or just "fn-literal". Something along those lines.

Reply:
I name it consistently with other predicate functions in this section:

```clojure
(defun clojure-ts--list-node-p (node)
  "Return non-nil if NODE is a Clojure list."
  (string-equal "list_lit" (treesit-node-type node)))

(defun clojure-ts--anon-fn-node-p (node)
  "Return non-nil if NODE is a Clojure anonymous lambda."
  (string-equal "anon_fn_lit" (treesit-node-type node)))
```

Reply:
Please, let me know if you still want me to rename it.

Reply:
I get this, but for me it's confusing, as `fn` lambdas are also anonymous functions when you think about it. Perhaps we can just extend the docstring to highlight what exactly does this match in terms of code.

Reply:
The docstring is currently `"Return non-nil if NODE is a Clojure function literal."`. Is it good enough?

Reply:
It will do for now.